src/routes/operator/+page.svelte

```diff
@@ -27,43 +27,111 @@
 
 	$: ({ running, config } = data);
 
+	let workerCountChartData = {
+		labels: [] as string[],
+		datasets: [
+			{
+				label: 'Order',
+				data: [] as number[],
+				borderColor: 'green',
+				tension: 0.1
+			},
+			{
+				label: 'Shipment',
+				data: [] as number[],
+				borderColor: 'blue',
+				tension: 0.1
+			},
+			{
+				label: 'Charge',
+				data: [] as number[],
+				borderColor: 'orange',
+				tension: 0.1
+			}
+		]
+	}
+
 	let completeChartData = {
-		labels: [],
+		labels: [] as string[],
 		datasets: [
 			{
-				label: 'Order Complete RPS',
-				data: [],
+				label: 'Order',
+				data: [] as number[],
 				borderColor: 'green',
 				tension: 0.1
 			},
+			{
+				label: 'Shipment',
+				data: [] as number[],
+				borderColor: 'blue',
+				tension: 0.1
+			},
+			{
+				label: 'Charge',
+				data: [] as number[],
+				borderColor: 'orange',
+				tension: 0.1
+			}
 		]
 	};
 
 	let backlogChartData = {
-		labels: [],
+		labels: [] as string[],
 		datasets: [
 			{
-				label: 'Order Backlog',
-				data: [],
-				borderColor: 'red',
+				label: 'Order',
+				data: [] as number[],
+				borderColor: 'green',
+				tension: 0.1
+			},
+			{
+				label: 'Shipment',
+				data: [] as number[],
+				borderColor: 'blue',
 				tension: 0.1
 			},
+			{
+				label: 'Charge',
+				data: [] as number[],
+				borderColor: 'orange',
+				tension: 0.1
+			}
 		]
 	};
 
 	let refreshInterval: number;
 
 	const updateChartData = async() => {
-		const response = await fetch('/api/order/stats');
-		const { completeRate, backlog } = await response.json();
-
 		const now = new Date().toLocaleTimeString();
 
+		workerCountChartData.labels = [...workerCountChartData.labels.slice(-20), now];
 		completeChartData.labels = [...completeChartData.labels.slice(-20), now];
 		backlogChartData.labels = [...backlogChartData.labels.slice(-20), now];
+
+		let response = await fetch('/api/order/stats');
+		let { workerCount, completeRate, backlog } = await response.json();
+
+		workerCountChartData.datasets[0].data = [...workerCountChartData.datasets[0].data.slice(-20), workerCount];
 		completeChartData.datasets[0].data = [...completeChartData.datasets[0].data.slice(-20), completeRate];
 		backlogChartData.datasets[0].data = [...backlogChartData.datasets[0].data.slice(-20), backlog];
-		chartData = chartData; // trigger reactivity
+
+		response = await fetch('/api/shipment/stats');
+		({ workerCount, completeRate, backlog } = await response.json());
+
+		workerCountChartData.datasets[1].data = [...workerCountChartData.datasets[1].data.slice(-20), workerCount];
+		completeChartData.datasets[1].data = [...completeChartData.datasets[1].data.slice(-20), completeRate];
+		backlogChartData.datasets[1].data = [...backlogChartData.datasets[1].data.slice(-20), backlog];
+
+		response = await fetch('/api/charge/stats');
```

Comment:
Can parallelize this with unless there's a reason not too.

```
const [orderStats, shipmentStats, chargeStats] = await Promise.all([fetch('/api/order/stats', '/api/shipment/stats', '/api/charge/stats'])
```

Reply:
All this a logic is gonna fail if any of the endpoints fails since it doesn't return workerCount, completeRate, backlog. Probably want to just return error instead of json from the endpoints and try/catch or return some defaults of { workerCount: 0, completeRate: 0, backlog: 0 }